src/commands/login/list-users-in-group.ts

```diff
@@ -60,7 +60,10 @@ export class ListUsersInGroup extends BaseCommand<typeof ListUsersInGroup> {
         'starting-token': lastEvaluatedKey,
       })
 
-    if ((lastEvaluatedKey || ListUsersInGroup.lastEvalutatedKeys) && !flags['no-input']) {
+    if (
+      (lastEvaluatedKey || ListUsersInGroup.lastEvalutatedKeys || ListUsersInGroup.pageNumber === 2) &&
+      !flags['no-input']
+    ) {
```

Comment:
thanks Youssef
do you mind making it a function with a explicit name to it ?

Reply:
thank you.
I was mostly thinking about having boolean variable for the multiple `if` conditions 😅 .
```diff
- if (
-      (lastEvaluatedKey || ListUsersInGroup.lastEvalutatedKeys || ListUsersInGroup.pageNumber === 2) &&
-       !flags['no-input']
- ) {
+ if (clearReasonOfThisIfBlock)
```
It's not only about this if.
Wdyt ?